www/topical-software.rst

```diff
@@ -58,126 +58,126 @@ Tutorials and texts
 ----------------------------------
 
-- `SWIG <http://www.swig.org/>`__: SWIG is a software development tool that connects programs written in C and C++ with a variety of high-level programming languages. SWIG is primarily used with common scripting languages such as Perl, Python, Tcl/Tk and Ruby.
+- `SWIG <http://www.swig.org/>`__: SWIG is a software development tool that connects programs written in C and C++ with a variety of high-level programming languages. SWIG is primarily used with common scripting languages, such as Perl, Python, Tcl/Tk, and Ruby.
 - `Boost.Python <http://www.boost.org/libs/python/doc/index.html>`__: a C++ library which enables seamless interoperability between C++ and Python. The `PythonInfo Wiki <https://wiki.python.org/moin/boost.python>`__ contains a good howto reference. "c++-sig": https://www.python.org/community/sigs/current/cplusplus-sig/ at python.org is devoted to Boost and you can subscribe to their mailing list.
-- `F2PY <https://sysbio.ioc.ee/projects/f2py2e/>`__: provides a connection between the Python and Fortran languages. F2PY is a Python extension tool for creating Python C/API modules from (handwritten or F2PY generated) signature files (or directly from Fortran sources).
+- `F2PY <https://sysbio.ioc.ee/projects/f2py2e/>`__: provides a connection between the Python and FORTRAN languages. F2PY is a Python extension tool for creating Python C/API modules from (handwritten or F2PY generated) signature files (or directly from FORTRAN sources).
 - `Cython <http://cython.org/>`__: allows the inclusion of C/C++ within Python code. It has facilities for automatic creation of C/C++ based Python extension modules, as well as for direct inlining of C/C++ code in Python sources. The latter combines the scripting flexibility of Python with the execution speed of compiled C/C++, while handling automatically all module generation details.
 - `Pyrex <http://www.cosc.canterbury.ac.nz/greg.ewing/python/Pyrex/>`__: Pyrex lets you write code that mixes Python and C data types any way you want, and compiles it into a C extension for Python. See also `Cython <http://cython.org>`__.
 - `PyCxx <http://cxx.sourceforge.net>`__: CXX/Objects is a set of C++ facilities to make it easier to write Python extensions. The chief way in which PyCXX makes it easier to write Python extensions is that it greatly increases the probability that your program will not make a reference-counting error and will not have to continually check error returns from the Python C API.
 - `ctypes <http://starship.python.net/crew/theller/ctypes>`__: a package to create and manipulate C data types in Python, and to call functions in dynamic link libraries/shared dlls. It allows wrapping these libraries in pure Python.
 - `railgun <http://tkf.bitbucket.org/railgun-doc/>`__: ctypes utilities for faster and easier simulation programming in C and Python
 
-Wrapping Matlab, R and IDL Codes
+Wrapping MATLAB, R, and IDL codes
 --------------------------------
```

Comment:
need one more `-` here, underline too short warning now

Reply:
Thanks for the careful review and apologies for the silly mistakes :-)